docs/CONTRIBUTING.md

```diff
@@ -29,9 +29,6 @@ fields. Here are some things you can write about your bug:
 -   What did happen instead?
 -   What is the version of the Cozy Stack?
 
-You can also use the [`cozy-stack tools bug`](cli/cozy-stack_bug.md) command to
-open the form to report issue prefilled with some useful system informations.
-
```

Comment:
Why did you remove this part?

Reply:
My mistake, I thought bug command was removed from cli as I couldn't find the documentation.
I rintroduced that paragraph and fixed he broken link.